ocaml/xapi-cli-protocol/tar_helpers.ml

```diff
@@ -0,0 +1,77 @@
+module Direct = struct
+  type 'a t = 'a
+  let return x = x
+  let ( >>= ) m f = f m
+end
+
+module Writer = struct
+  type out_channel = Unix.file_descr
+  type 'a t = 'a Direct.t
+  (* XXX: there's no function to write directly from a bigarray *)
+  let really_write (fd : out_channel) buffer =
+    let s = Cstruct.to_string buffer |> Bytes.of_string in
+    if Bytes.length s > 0
+    then ignore (Unix.write fd s 0 (Bytes.length s))
+end
+module TarH = Tar.HeaderWriter(Direct)(Writer)
+
+module TarHelper : sig
```

Comment:
We now have `Tar_helpers.TarHelper`. Maybe introduce `tar_helpers.mli` and avoid a sub module here.